### PR TITLE
Named parameter in visit

### PR DESCRIPTION
**Pull request by jillianrosile**

### CODE REVIEW

File: lib/gamera/modules/visitable.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ module Visitable
     #
     # @param fail_on_redirect [Boolean] Whether to fail if the site redirects to a page that does not match the url_matcher regex
     # @raise [WrongPageVisited] if the site redirects to URL that doesn't match the url_matcher regex and fail_on_redirect is true
-    def visit(fail_on_redirect = true)
+    def visit(fail_on_redirect: true)
```
Comment (jasonnoble) [inline lib/gamera/modules/visitable.rb:11]: Seems like this change should go in a `CHANGES.md` or at least a readme?  Then again, it's pre-beta software.

Comment (jillianrosile) [inline lib/gamera/modules/visitable.rb:11]: Reminded me to update the README example, thanks. We probably do want a `CHANGES.md` eventually but I'm not sure this is in wide enough use yet to warrant it.

File: spec/unit/page_spec.rb [hunk 2]
```diff
@@ -53,7 +53,7 @@ class TextNotFoundError < StandardError; end
 
   it 'ignores a redirect if told to' do
     redirect_page = RedirectPage.new
-    expect { redirect_page.visit(false) }.not_to raise_error
+    expect { redirect_page.visit(fail_on_redirect: false) }.not_to raise_error
```
Comment (asimy) [inline spec/unit/page_spec.rb:56]: I'd suggest checking for the specific WrongPageVisited error.

**Comment by asimy** [comment]
Nitpick aside, :+1:

**Pull request merged**